examples/credit-risk-end-to-end/04_Credit_Risk_Model_Serving.ipynb

```diff
@@ -0,0 +1,697 @@
+       "      <td>0&lt;=X&lt;200</td>\n",
+       "      <td>24.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>for free</td>\n",
+       "      <td>44.0</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>1.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>764</td>\n",
+       "      <td>2463.0</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>no checking</td>\n",
+       "      <td>24.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>own</td>\n",
+       "      <td>27.0</td>\n",
+       "      <td>3.0</td>\n",
+       "      <td>2.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>504</td>\n",
+       "      <td>1207.0</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>&lt;0</td>\n",
+       "      <td>24.0</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>rent</td>\n",
+       "      <td>24.0</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>1.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    ID  credit_amount  installment_commitment checking_status  duration  \\\n",
+       "0   18        12579.0                     4.0        0<=X<200      24.0   \n",
+       "1  764         2463.0                     4.0     no checking      24.0   \n",
+       "2  504         1207.0                     4.0              <0      24.0   \n",
+       "\n",
+       "   num_dependents   housing   age  residence_since  existing_credits  \n",
+       "0             1.0  for free  44.0              2.0               1.0  \n",
+       "1             1.0       own  27.0              3.0               2.0  \n",
+       "2             1.0      rent  24.0              4.0               1.0  "
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "records.head(3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd828758-6c57-4f9e-bbda-3983b6579da2",
+   "metadata": {},
+   "source": [
+    "### Get Predictions from the Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f446d7ec-0dae-409a-82a2-c0d7016c2001",
+   "metadata": {},
+   "source": [
+    "Now we can request predictions from our trained model. \n",
+    "\n",
+    "For convenience, we output the predictions along with the implied loan designations. Remember that these are predictions on loan outcomes, given context data from the loan application process. Since we have access to the actual `class` outcomes, we display those as well to see how the model did.|"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "70203f7b-f1e5-46ba-8623-f10bf3a5abf8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get predictions from the model\n",
+    "preds = model.predict(records)"
```

Comment:
The only nit I have here is that probably it'd be good to have the model serving to be a separate endpoint, but we can leave that for another time.